apps/explorer/src/components/IotaTokenCard.tsx

```diff
@@ -3,20 +3,21 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Panel } from '@iota/apps-ui-kit';
-import { COIN_GECKO_IOTA_URL, CoinIcon, ImageIconSize, useIotaCoinData } from '@iota/core';
+import {
+    COIN_GECKO_IOTA_URL,
+    CoinIcon,
+    formatBalanceToUSD,
+    ImageIconSize,
+    useIotaCoinData,
+} from '@iota/core';
 import { ButtonOrLink } from '~/components/ui';
 import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
 
 export function IotaTokenCard(): JSX.Element {
     const { data } = useIotaCoinData();
```

Comment:
I see that `useIotaCoinData` use a bad endpoint for apps-backend. We are fetching on `coins/iota` and the correct endpoint is `coin-price/iota`. Then the response is only `{ price: Number }`
```suggestion
    const { data } = useIotaCoinData();
```

Reply:
do in https://github.com/iotaledger/iota/issues/4965